controllers/suite_test.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*
+
+
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Do we need two licence headers here?

Reply:
Was this file autogenerated too?

Reply:
This was autogenerated but I do belive we should remove the extra license and only keep the Red Hat one that we have on other files, there might be some other files as well with this kind of two license I'll remove the extra one from them as well :)

Reply:
Thanks, I have noticed just this one occurrence.
Btw do we need this file at all? It looks like it is used for Ginkgo tests which AFAIK we don't use.

Reply:
These tests are a nice way to test the controllers. We will need in the long run.